kentikapi/cloud_exports_integration_test.go

```diff
@@ -509,12 +507,11 @@ func TestClient_CreateCloudExport(t *testing.T) {
 			server.Start()
 			defer server.Stop()
 
-			client, err := kentikapi.NewClient(kentikapi.Config{
-				CloudExportHostPort: server.url,
-				AuthToken:           dummyAuthToken,
-				AuthEmail:           dummyAuthEmail,
-				DisableTLS:          true,
-			})
+			client, err := kentikapi.NewClient(
+				kentikapi.WithAPIURL("http://"+server.url),
+				kentikapi.WithAuthEmail(dummyAuthEmail),
+				kentikapi.WithAuthToken(dummyAuthToken),
```

Comment:
not sure here, shouldn't it be always both email+token or nothing? Like user+password?

Reply:
I think Your right. If Daniel have nothing against I will merge those two options.

Reply:
Done